docs/assets/highlight.css

```diff
@@ -1,15 +1,19 @@
 :root {
-    --light-code-background: #FFFFFF;
-    --dark-code-background: #1E1E1E;
+    --light-code-background: #ffffff;
```

Comment:
@rugoncalves was this because of eslint/pretier? Or is there another reason?

Reply:
I didn't change this manually. It's like this in the [main branch](https://github.com/OutSystems/outsystems-datagrid/blob/00d4fcf238c6897b1cabe3ee00ffc7667293298e/docs/assets/highlight.css#L2).